.github/buildomat/jobs/package.sh

```diff
@@ -6,6 +6,7 @@
 #: rust_toolchain = "nightly-2022-09-27"
 #: output_rules = [
 #:	"=/work/package.tar.gz",
+#:	"=/work/zones/*.tar.gz",
```

Comment:
There is a single tarball that currently is just a `.tar`, and not a `.tar.gz` -- sled agent itself.

We might wanna grab that one too, since we'll need it when constructing the Helios image?

Reply:
There's also `out/maghemite.tar`. I figured I'd deal with both of those later, it's simple enough to tweak this pair of scripts.